spring-cloud-gcp-data-firestore/src/test/java/com/google/cloud/spring/data/firestore/FirestoreTemplateTests.java

```diff
@@ -215,6 +215,23 @@ public void deleteTest() {
 		verify(this.firestoreStub, times(1)).commit(eq(builder.build()), any());
 	}
 
+	@Test
+	public void deleteByIdTest() {
+		mockCommitMethod();
+		Flux<String> idPublisher = Flux.just("e1", "e2");
+		StepVerifier.create(
+				this.firestoreTemplate
+						.deleteById(idPublisher, TestEntity.class))
+				.verifyComplete();
+
+		CommitRequest.Builder builder = CommitRequest.newBuilder()
+				.setDatabase("projects/my-project/databases/(default)");
+
+		builder.addWrites(Write.newBuilder().setDelete(parent + "/testEntities/e1").build());
+		builder.addWrites(Write.newBuilder().setDelete(parent + "/testEntities/e2").build());
+
+		verify(this.firestoreStub, times(1)).commit(eq(builder.build()), any());
```

Comment:
`times(1)` is the default value, so it can be omitted -- https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html
(although I keep having to re-search for this documentation because I don't seem to trust it for some reason)